pinot-core/src/main/java/org/apache/pinot/core/common/DataBlockCache.java

```diff
@@ -493,6 +493,24 @@ public int[] getNumValuesForMVColumn(String column) {
     return numValues;
   }
 
+  /**
+   * Get the bytes values for a multi-valued column.
+   *
+   * @param column Column name
+   * @return Array of bytes values
+   */
+  public byte[][][] getBytesValuesForMVColumn(String column) {
```

Comment:
(minor) Move it in front of `getNumValuesForMVColumn()`

Reply:
Done